examples/substrate-react-example/src/UserInfo.tsx

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable */
+// @ts-nocheck
```

Comment:
please don't use `@ts-ignore` and `@ts-nocheck` or any other thing to get around type checkers and liters